asapdiscovery-alchemy/asapdiscovery/alchemy/utils.py

```diff
@@ -338,6 +326,25 @@ def cancel_actioned_tasks(self, network_key: ScopedKey) -> list[ScopedKey]:
             canceled_tasks = []
         return canceled_tasks
 
+    def adjust_weight(self, network_key: ScopedKey, weight: float) -> float:
```

Comment:
Return type is a tuple

```suggestion
    def adjust_weight(self, network_key: ScopedKey, weight: float) -> Tuple[float, float]:
```